### DISCUSSION TITLE
我想将放大后的视频导出并保存，主要用于在手机上查看，请教下我该怎么弄

### DISCUSSION THREAD

**Discussion by jenfonro** [hooke007/mpv_PlayKit#416] [mpv 通用]
我想将放大后的视频导出并保存，主要用于在手机上查看，请教下我该怎么弄

**Comment by hooke007** [comment]
从头写一个编码教程超出了我的能力和兴趣。。。

除了我自己写的一部分，其它的建议自学
https://mpv.io/manual/master/#encoding
https://ffmpeg.org/ffmpeg-all.html
http://www.vapoursynth.com/doc/
https://github.com/vcb-s/guides
